### PR TITLE
Changed settings

### PR DESCRIPTION
**Pull request by giovannivolpe**
- [] complete methods .getAvailableSettings() in all measures
- [] add test in test_Measure
- [] add comments in code already commented

### CODE REVIEW

**Comment by egolol** [comment]
@giovannivolpe commited the changes

File: braph/graph/measures/DegreeAv.m [hunk 1]
```diff
@@ -90,6 +88,9 @@
                 'Connection weights are ignored in calculations.' ...
                 ];
         end
+        function available_settings = getAvailableSettings()
```
Comment (giovannivolpe) [inline braph/graph/measures/DegreeAv.m:91]: add comment

File: braph/graph/measures/Eccentricity.m [hunk 2]
```diff
@@ -97,6 +96,19 @@
...
+        function available_settings = getAvailableSettings()
+            % GETAVAILABLESETTINGS returns the setting available to Eccentricity
+            %
+            % AVAILABLESETTINGS = GETAVAILABLESETTINGS() returns the
+            % settings available to Eccentricity.
+            % ECCENTRICITYRULE = 'default' (default) - calculates ECCENTRICITY of global graph.
+            %                    'subgraphs' - calculates ECCENTRICITY within connected subgraphs.
+            % 
+            % See also getCompatibleGraphList()
+            available_settings = {
```
Comment (giovannivolpe) [inline braph/graph/measures/Eccentricity.m:108]: leave space

File: braph/graph/measures/InEccentricity.m [hunk 3]
```diff
@@ -98,6 +96,18 @@
...
         end
+        function available_settings = getAvailableSettings()
+            % GETAVAILABLESETTINGS returns the setting available to InEccentricity
+            %
+            % AVAILABLESETTINGS = GETAVAILABLESETTINGS() returns the
+            % settings available to InEccentricity.
+            % ECCENTRICITYRULE = 'default' (default) - calculates INECCENTRICITY of global graph.
+            %                    'subgraphs' - calculates INECCENTRICITY within connected subgraphs.
+            % 
+            available_settings = {
```
Comment (giovannivolpe) [inline braph/graph/measures/InEccentricity.m:107]: space

**Comment by egolol** [comment]
@giovannivolpe  updated

**Pull request merged**